### PR TITLE
Wk 23 filter panel

### PR DESCRIPTION
**Pull request by BennievanderWel**

### CODE REVIEW

File: src/assets/icons/Chevron-Top.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 53.1 (72631) - https://sketchapp.com -->
+    <title>Icon/32/Chevron-Top   </title>
+    <desc>Created with Sketch.</desc>
+    <g id="Icon/32/Chevron-Top---" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```
Comment (RRMoelker) [inline src/assets/icons/Chevron-Top.svg]: id

File: src/assets/icons/Chevron-Top.svg [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        <polygon id="Path" fill="#000000" points="29.0899743 25.7568858 16 12.6771943 2.91002571 25.7568858 0 22.8571429 16 6.85714286 32 22.8571429"></polygon>
```
Comment (RRMoelker) [inline src/assets/icons/Chevron-Top.svg]: id

File: src/assets/icons/icon-filter.svg [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 54.1 (76490) - https://sketchapp.com -->
+    <title>Group 7</title>
+    <desc>Created with Sketch.</desc>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```
Comment (RRMoelker) [inline src/assets/icons/icon-filter.svg]: Nog meer ids

File: src/components/filterPanel/FilterPanel.js [hunk 4]
```diff
@@ -240,7 +251,22 @@ const FilterPanel = ({
   }
 
   return (
-    <div className={styles.FilterPanel}>
+    <div
+      className={classNames(
+        styles.FilterPanel,
+        showPanel ? '' : styles.FilterPanelCollapsed
```
Comment (RRMoelker) [inline src/components/filterPanel/FilterPanel.js]: the classNames helper is used for exactly this situation right? Where you optionally add a class. No ternary needed..?

File: src/shared/selectMenu/SelectMenu.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    return (
+      <div
+        className={classNames(styles.Menu, !showMenu ? styles.MenuHide : '')}
+      >
+        {items.map((i, index) => (
+          <button
+            key={index}
+            className={styles.Option}
+            onClick={() => {
+              i.onClick();
```
Comment (RRMoelker) [inline src/shared/selectMenu/SelectMenu.js]: Given that items is some special array an propType specification would be pleasant.

File: src/shared/selectMenu/SelectMenu.test.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import SelectMenu from './SelectMenu';
+
+describe('SelectMenu', () => {
```
Comment (RRMoelker) [inline src/shared/selectMenu/SelectMenu.test.js]: These test are really bare bone.

File: src/shared/selectMenu/SelectMenu.js (hunk 5 above)
Comment (BennievanderWel) [reply src/shared/selectMenu/SelectMenu.js]: Yes, true. Added it.

File: src/components/filterPanel/FilterPanel.js (hunk 4 above)
Comment (BennievanderWel) [reply src/components/filterPanel/FilterPanel.js]: This is a habit I have, but you are totally right. I changed it.

**Pull request merged**